AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1611,15 +1610,6 @@ - (void)animateLayoutTransition:(id<ASContextTransitioning>)context
       CGRect fromFrame = [context initialFrameForNode:subnode];
       CGRect toFrame = [context finalFrameForNode:subnode];
       if (CGSizeEqualToSize(fromFrame.size, toFrame.size) == NO) {
-        // To crossfade resized subnodes, show a snapshot of it on top.
-        // The node itself can then be treated as a newly-inserted one.
-        UIView *snapshotView = [subnode.view snapshotViewAfterScreenUpdates:YES];
```

Comment:
I think even if we remove this entirely, it is better than before -- however, it would be great to support this as an option because there are some nice use cases.  How about including it in the same place that the upward-propagating "subnode PREVENTS ASCellNode from animation" option?  @nguyenhuy would know the name for this.  But, the idea would be, by default there are no snapshots...unless one subnode declares that it requires snapshots to transition, in which case the entire transition would be snapshotted?

Alternatively, we could have every node expose a .shouldSnapshotTransitions BOOL, and do some kind of performoneverysubnode call here.